src/answer/answer.service.ts

```diff
@@ -498,4 +631,23 @@ export class AnswerService {
       userId,
     );
   }
+
+  async getAnswerCount(userId: number | undefined): Promise<number> {
+    if (userId == undefined) return 0;
```

Comment:
weird. the semantic is not clear.

Reply:
The user is not logged in, so the concept of number of responses does not exist, rather than the user's number of responses being equal to 0.

Reply:
so the method should return undefined as well, rather than 0

Reply:
fixed: https://github.com/SageSeekerSociety/cheese-backend/commit/51777d4ee22f5bd07ce71ee33c9b9c3bc29af630